core/controllers/editor.py

```diff
@@ -197,6 +188,34 @@ def delete(self, exploration_id):
 class ExplorationRightsHandler(EditorHandler):
     """Handles management of exploration editing rights."""
 
+    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+
+    @acl_decorators.can_play_exploration
+    def get(self, exploration_id):
+        """Gets the user rights for an exploration."""
```

Comment:
As stated above, I think we should create a separate handler for this, as the handler name is ExplorationRights and for the put request it changes ExplorationRights and it's exploration dependent so we expect it should return the explorationRights object in the GET request. (Maybe we should go with UserExplorationPermissionsHandler or UserActivityPermissionHandler)

Reply:
I've created a new handler for this. PTAL @DubeySandeep